File: api-tester-java/src/main/java/com/revenuecat/apitesterjava/OfferingsAPI.java
package com.revenuecat.apitesterjava;

import com.revenuecat.purchases.Offering;
import com.revenuecat.purchases.Offerings;

import java.util.Map;

@SuppressWarnings({"ConstantConditions", "unused"})
final class OfferingsAPI {
    static void check() {
        final Offerings offerings = null;

        final Offering current = offerings.getCurrent();
        final Map<String, Offering> all = offerings.getAll();
        final Offering o1 = offerings.getOffering("");
        final Offering o2 = offerings.get("");
    }
}

File: api-tester-kotlin/src/main/java/com/revenuecat/apitesterkotlin/OfferingsAPI.kt
package com.revenuecat.apitesterkotlin

import com.revenuecat.purchases.Offering
import com.revenuecat.purchases.Offerings

@Suppress("unused")
private class OfferingsAPI {
    fun check() {
        val offerings: Offerings? = null

        val current: Offering? = offerings!!.current
        val all: Map<String, Offering> = offerings.all
        val o1 = offerings.getOffering("")
        val o2: Offering? = offerings[""]
    }
}
